src/test/unit/utils/DOMUtils.test.ts

```
@@ -0,0 +1,263 @@
+    (global as any).HTMLElement = dom.window.HTMLElement;
+    (global as any).getComputedStyle = dom.window.getComputedStyle;
+    
+    sandbox = sinon.createSandbox();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+    // クリーンアップ
+    delete (global as any).document;
+    delete (global as any).window;
+    delete (global as any).HTMLElement;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)